### DISCUSSION TITLE
Is mintty on W11 supported? It hasn't worked for a while.

### DISCUSSION THREAD

**Discussion by ykhan21** [sxyazi/yazi#1969] [1. Q&A]
### What system are you running Yazi on?

Windows

### What terminal are you running Yazi in?

mintyy 3.7.6

### `yazi --debug` output

```Shell
Yazi
    Version: 0.3.3 (7c445ce 2024-09-04)
    Debug  : false
    OS     : windows-x86_64 (windows)

Ya
    Version: 0.3.3 (7c445ce 2024-09-04)

Emulator
    Emulator.via_env: ("xterm", "mintty")
    Emulator.via_csi: Ok(Unknown([]))
    Emulator.detect : Mintty

Adapter
    Adapter.matches: Iterm2

Desktop
    XDG_SESSION_TYPE           : None
    WAYLAND_DISPLAY            : None
    DISPLAY                    : Some("needs-to-be-defined")
    SWAYSOCK                   : None
    HYPRLAND_INSTANCE_SIGNATURE: None
    WAYFIRE_SOCKET             : None

SSH
    shared.in_ssh_connection: false

WSL
    WSL: false

Variables
    SHELL              : Some("C:\\Program Files\\Git\\usr\\bin\\bash.exe")
    EDITOR             : Some("focus")
    VISUAL             : None
    YAZI_FILE_ONE      : Some("C:\\Program Files\\Git\\usr\\bin\\file.exe")
    YAZI_CONFIG_HOME   : None

Text Opener
    default: Some(Opener { run: "code %*", block: false, orphan: true, desc: "code", for_: None, spread: true })
    block  : Some(Opener { run: "code -w %*", block: true, orphan: false, desc: "code (block)", for_: None, spread: true })

Multiplexers
    TMUX               : false
    tmux version       : program not found
    ZELLIJ_SESSION_NAME: None
    Zellij version     : program not found

Dependencies
    file             : 5.45
    ueberzugpp       : program not found
    ffmpegthumbnailer: program not found
    magick           : 7.1.1-38
    fzf              : 0.55.0
    fd               : 10.2.0
    rg               : 14.1.1
    chafa            : 1.14.4
    zoxide           : program not found
    7z               : 24.08
    7zz              : program not found
    jq               : 1.7.1


--------------------------------------------------
When reporting a bug, please also upload the `yazi.log` log file - only upload the most recent content by time.
You can find it in the "C:\\Users\\USERNAME\\AppData\\Roaming\\yazi\\state" directory.
```

### Describe the question

yazi image previews don't work in the Git Bash mintty terminal.
This issue was closed and not resolved for the same concern half a year ago.
https://github.com/sxyazi/yazi/issues/974

### Validations

- [X] I have searched the existing discussions/issues

**Comment by sxyazi** [comment]
The last time I tested it, after some hacks, I managed to get Mintty to display images. 

But if it's not working now, I wouldn't be surprised because ConPTY interferes with the communication between Yazi and Mintty in various ways (See https://github.com/mintty/mintty/issues/1228), leading to potential image rendering issues. Any changes on either side (Yazi, Mintty, or ConPTY) could cause these hacks to stop working.

The good news, though, is that Windows Terminal devs are testing a new version of ConPTY, in which all VT sequences are passed through to the client terminal as is (See https://github.com/sxyazi/yazi/issues/1557#issuecomment-2340294607), and once it's released and with future Windows system updates, Yazi will be able to use Sixel with Mintty
